src/main/flight/gps_rescue.c

```diff
@@ -41,14 +41,14 @@
 #include "fc/runtime_config.h"
 
 #include "flight/failsafe.h"
-#include "flight/imu.h"
+#include "flight/imu.h" // for yaw attitude values
```

Comment:
```suggestion
#include "flight/imu.h"
```

Reply:
Ok, Ok ... these were added because I was checking the necessity for each one, and while checking I discovered exactly why they were there.
Removed.